src/test/java/assertions/PersonTest.java

```diff
@@ -14,21 +16,37 @@
 
     @Test
     public void getFullNameReturnsFirstnameSpaceLastname(){
-        // TODO implement
-        throw new NotImplementedException();
+        Person p = new Person("Harry", "Moser", LocalDate.now());
+
+        String fullname = p.getFullName();
+
+        Assert.assertEquals("Harry Moser", fullname);
     }
 
     // TODO some more useful tests
+    @Test
+    public void getAgeReturnTestObObjektPeriodeZurckKommt(){
+        Person t = new Person("Harry", "Moser", LocalDate.now());
+        Period  period = t.getAge();
+        Assert.assertNotNull(period);
+
+    }
+    @Test
+    public void getAgeRetunObAlterRichtigRechnet() {
+        Person p = new Person("Harry", "Moser", LocalDate.of(1973, 04, 10));
+        Period  period = p.getAge();
+        Assert.assertEquals(43,period.getYears());
+
 
-    // getAge
 
+    }
 
     // TODO verbessern. Hinweis: Repeatable (wiederholbar) zu jeder Zeit.
     @Test
     public void getAgeReturns10YearsIf2006() throws Exception {
```

Comment:
PS: In der [README Datei](https://github.com/ibwgr/junit-exercises/blob/master/README.md) siehst du die git Befehle um die neusten Änderungen von mir zu bekommen.